source/docs/yearly-overview/removed-features.rst

```diff
@@ -0,0 +1,69 @@
+# Removed features
+
+## Labview
+NI will not support Labview on SystemCore.
+
+## Java tools
+
+All Java GUI tools have been removed due to lack of maintenance.
+
+### Shuffleboard
+Use Elastic as a driver dashboard, use AdvantageScope for debugging and data visualization.
+
+### SmartDashboard
+Use Elastic as a driver dashboard, use AdvantageScope for debugging and data visualization.
+
+### RobotBuilder
+No replacement. There are many upcoming codegen projects however.
+
+### PathWeaver
+Use Choreo or PathPlanner instead.
+
+## roboRIO Team Number Setter
+Config for SystemCore will be handled via the web interface.
+
+## Hardware
+
+### Relay
+No support.
```

Comment:
I'd say something like SystemCore doesn't support X for all the hardware items below